src/LSST/system/TelemetryCom.cpp

```diff
@@ -197,51 +208,120 @@ void TelemetryCom::ServerConnectionHandler::_servConnHandler() {
             // calling send().
             ssize_t status = send(_servConnHSock, msg.c_str(), msg.length(), 0);
             LTRACE("TelemetryCom send status=", status, " msg=", msg);
+            if ((msgSentCount++)%1000 == 0) {
```

Comment:
Please help to put the magic number of `1000` into a named constant and explain it.